audited.gemspec

```diff
@@ -16,11 +16,11 @@ Gem::Specification.new do |gem|
 
   gem.required_ruby_version = ">= 2.3.0"
 
-  gem.add_dependency "activerecord", ">= 5.2", "< 8.1"
-  gem.add_dependency "activesupport", ">= 5.2", "< 8.1"
+  gem.add_dependency "activerecord", ">= 5.2", "< 8.0"
```

Comment:
nooo rip. that means when rails 8.0 is out i will have to fork this project again to be able to use rails main branch. i thought i had more time 😭 

honestly i don't know why there is a restriction on the upper bound
it should be like this `gem.add_dependency "activerecord", ">= 5.2"`

Reply:
I don't really have a strong opinion here. I see this pattern of limiting the upper bound a lot, but is it really helping more than hurting?

Reply:
My goal for this PR was to get CI passing and confirm Rails 7.2 compatibility. I removed the rails80 appraisal because it was not working. Without a passing test suite for 8.0, I didn't feel comfortable allowing 8.0 in the gemspec, and I don't have a real app running on Rails `main` that I could manually test.

Rubygems itself frowns upon unbounded dependencies. I don't know how closely their recommendations are followed, but the Rubygems validation code emits a warning for it:

https://github.com/rubygems/rubygems/blob/b22914018e9c5f7b5b31eab6fa39ea4e82c623dc/lib/rubygems/specification_policy.rb#L222-L235

For `audited` specifically, this is a mission-critical gem, it can result in data loss if it doesn't work correctly, and it hooks deeply into ActiveRecord. So I would be uncomfortable if it allowed versions of Rails that were not thoroughly tested. So I lean toward being more strict in the gemspec.

That said, ultimately if this is a very-lightly maintained gem, and the philosophy is "use it at your own risk", then an unbounded requirement makes sense. On the other hand, if there is a plan for actively maintaining and testing every release of Rails and Ruby versions going forward, including prereleases, then I think a more strict requirement would work.